D.py

```diff
@@ -0,0 +1,2 @@
+def godmornig():
```

Comment:
```suggestion
def goodmorning():
```

godmornid → goodmorning